docs/application/dotnet/guides/nui/nui-components/Pagination.md

```diff
@@ -0,0 +1,163 @@
+# Pagination
+
+Pagination is a common component supporting multiple pages. It shows the number of pages available and the currently active page.
+
+![Pagination](./media/Pagination.PNG)
+
+## Create with property
+
+To create a Pagination using property, follow these steps:
+
+1. Create Pagination using the default constructor:
+
+    ```cs
+    pagination = new Pagination();
+    ```
+
+2. Set the Pagination property:
+
+    ```cs
+    var indicatorImageUrlStyle = new PaginationStyle()
+    {
+        IndicatorImageUrl = new Selector<string>
+        {
+            Normal = "pagination_ic_nor.png",
+            Selected = "pagination_ic_sel.png"
+        }
+    };
+    pagination.ApplyStyle(indicatorImageUrlStyle);
+    pagination.IndicatorSize = new Size(26, 26);
+    pagination.IndicatorSpacing = 8;
+    pagination.Name = "Pagination1";
+    pagination.Size = new Size(300, 50);
+    pagination.BackgroundColor = new Color(1.0f, 1.0f, 1.0f, 0.6f);
+    pagination.IndicatorCount = 5;
+    pagination.SelectedIndex = 1;
+    root.Add(pagination);
+    ```
+
+Following output is generated when the Pagination is created using property:
+
+![Pagination1](./media/Pagination1.PNG)
```

Comment:
The image is broken. I think the mismatch in the extension is the reason. Would you plase use small letter in the image name also?

Pagination1.png

Reply:
> The image is broken. I think the mismatch in the extension is the reason. Would you plase use small letter in the image name also?
> 
> Pagination1.png

Fixed